openpype/hosts/blender/plugins/publish/validate_mesh_has_uv.py

```diff
@@ -3,18 +3,19 @@
 import bpy
 
 import pyblish.api
-import openpype.hosts.blender.api.action
+from openpype.api import ValidateContentsOrder
+from openpype.hosts.blender.api.action import SelectInvalidAction
 
 
 class ValidateMeshHasUvs(pyblish.api.InstancePlugin):
     """Validate that the current mesh has UV's."""
 
-    order = pyblish.api.ValidatorOrder
+    order = ValidateContentsOrder
     hosts = ["blender"]
     families = ["model"]
-    category = "geometry"
+    category = "uv"
```

Comment:
This remark isn't necessarily about this change - it's more a general comment regarding the `category` attribute.

I believe `category` is actually an unused attribute across the tooling. Unless the new publishers use this? I'm wondering whether we should just clean up all these `category` definitions.

@mkolar do you know more?

Reply:
To be honest I personally completely forgot this was there, and I agree that it should be investigated and tackled across the board. It doesn't hurt here, but doesn't really add anything either.

Reply:
Ok, I leave the code as it is for `category = "geometry"` attributes in validators if there are consistent, but removed it for inconsistent as in camera validators.